aula-pratica-02/ex2.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+#include<stdlib.h>
+int primo(int n);
+
+int primo(int n)

```

Comment:
it's  probably better to return a `bool` from this function since it always return or `1` or `0`. And also, as the function always return a true or false (or a 0 or 1), I'd rename the function to explicitly tell the reader that this returns a true/false value, as a good practice, I always see an `isSomething` in the name of the function like this:

```suggestion
bool is_prime(int n)
```
(I also changed the name of the function from portuguese to english because of personal taste, and because writing `e_primo` is strange :stuck_out_tongue:)

and where you return 0, you change to `return false` and return 1 to `return true`

```diff
-return 0;
+return false;
```

```diff
-return 1;
+return true;
```